src/code/z_actor.c

```diff
@@ -90,7 +90,7 @@ void ActorShadow_DrawFoot(GlobalContext* globalCtx, Light* light, MtxF* arg2, s3
 
     gSPMatrix(POLY_OPA_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_actor.c", 1687),
               G_MTX_MODELVIEW | G_MTX_LOAD);
-    gSPDisplayList(POLY_OPA_DISP++, &D_04048180);
+    gSPDisplayList(POLY_OPA_DISP++, &gFootShadowDL);
```

Comment:
Ideally, all `&` should be removed when switching to the right symbols/types for arrays (eg. display lists)